From: uazo <uazo@users.noreply.github.com>
Date: Tue, 27 Feb 2024 08:51:40 +0000
Subject: Disable offline autofetch flag

Disable auto fetch of content when browser is online

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 .../common/chrome_features_cc/Disable-offline-autofetch.inc  | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 Spectre Browser_flags/chrome/common/chrome_features_cc/Disable-offline-autofetch.inc

diff --git a/Spectre Browser_flags/chrome/common/chrome_features_cc/Disable-offline-autofetch.inc b/Spectre Browser_flags/chrome/common/chrome_features_cc/Disable-offline-autofetch.inc
new file mode 100644
--- /dev/null
+++ b/Spectre Browser_flags/chrome/common/chrome_features_cc/Disable-offline-autofetch.inc
@@ -0,0 +1,5 @@
+#if BUILDFLAG(IS_ANDROID)
+
+SET_Spectre Browser_FEATURE_DISABLED(kOfflineAutoFetch);
+
+#endif
--
